src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -765,7 +776,7 @@ def _unknown_error(self, error):
         )
 
     def _bn_to_int(self, bn):
-        assert bn != self._ffi.NULL
+        self.openssl_assert(bn != self._ffi.NULL)
```

Comment:
This is an internal invariant isn't it? Not checking the result of an OpenSSL call